### ISSUE TITLE
[A11y] Edit User - Status Messages - Failure

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1861]
### Operating System

Windows

### Browser

Edge

### Affected page

Supporting (/contact, /help, etc)

### Admin user?

Yes

### Issue Description

On a failed submit, it's not obvious to assistive tech users that the form had errors.  A screen-reader user, for instance, has to read a fair amount of the page's content to even determine that anything went wrong.

### Suggested Fix

- The page title should indicate that there was an error (e.g., "There were X errors -- Update Registration -- Oregon Digital")
- The `error_explanation` div should get focus about 1-2 seconds after the DOM is loaded so it's the first thing a screen reader hears.

This is deque's recommended remediation - the delay is supposed to help screen readers have a chance to parse the page.  In practice I'm not sure if that's a good idea as it seems like it could cause confusion for other users who have already put the focus into the field that needs to be fixed.  This is probably a question for people with more expertise.

**Comment by jechols** [comment]
QA likely requires a screenreader, at least for making sure the focus is moved and/or the status is read out

**Comment by jechols** [comment]
Upon further review, I'm finding that this is a nice-to-have, not a WCAG failure. The error is currently identified in a heading, which isn't ideal, but given time constraints it's acceptable.
